src/WizardsWardrobeGui.lua

```diff
@@ -1895,3 +1899,44 @@ function WWG.StartAlphaAnimation( control, duration, startAlpha, endAlpha )
 	end
 	return animation, timeline
 end
+
+function WWG.ShowExitWarning()
+	local menu = WINDOW_MANAGER:GetControlByName("ZO_GameMenu_InGame").gameMenu
+	local logoutControl
+	local quitControl
+
+	for i, children in ipairs(menu.navigationTree.rootNode.children) do
+		if children.control:GetText() == string.upper(GetString(SI_GAME_MENU_LOGOUT)) then
+			logoutControl = children.control
+		elseif children.control:GetText() == string.upper(GetString(SI_GAME_MENU_QUIT)) then
+			quitControl = children.control
+		end
+	end
```

Comment:
None of this is necessary if the feature is disabled, or if the warnings are created since you can get the parent from them.
You should generally endeavour to run the minimum required code.